### PR TITLE
CI: check env in wheel uploader

### PR DESCRIPTION
**Pull request by andyfaff**

### CODE REVIEW

**Comment by andyfaff** [comment]
should fix #23182

File: tools/ci/cirrus_general.yml [hunk 1]
```diff
@@ -90,16 +83,28 @@ wheels_upload_task:
...
-    upload_wheels
+        # The name of the zip file is derived from the `wheels_artifact` line.
+        # If you change the artifact line to `myfile_artifact` then it would be
+        # called myfile.zip
+        
+        curl https://api.cirrus-ci.com/v1/artifact/build/$CIRRUS_BUILD_ID/wheels.zip --output wheels.zip
+        unzip wheels.zip
+        
+        source ./tools/wheels/upload_wheels.sh
+        # IS_PUSH takes precedence over IS_SCHEDULE_DISPATCH
```
Comment (seberg) [inline tools/ci/cirrus_general.yml:105]: Is this comment still relevant?

**Review by seberg — APPROVED** [review]
LGTM just a question about comment being relevant.

**Comment by seberg** [comment]
Thanks, lets get it in, since it unbreaks CI.

**Pull request merged**